src/Preprocesser.js

```diff
@@ -1,16 +1,15 @@
-const natural = require("natural");
 const WordPos = require("wordpos");
 const WeightedGraph = require('./WeightedGraph').WeightedGraph;
 
 class Preprocesser{
 	constructor(){
-		this.tokenizer = new natural.SentenceTokenizer(); 
+		this.tokenizer = require('sbd');
```

Comment:
Should this go in the header? Seems like a style thing but I guess there's no ESlint

Reply:
Yeah i wrote this code a really long time ago :/